app/views/categories/show.html.erb

```diff
@@ -13,7 +13,11 @@
     <% end %>
         <h1 class="page-title"><%= title @category.label %></h1>
         <div class="details">
-          <%= sanitize @category.long_description.html_safe unless @category.long_description.nil? %>
+          <% unless @category.long_description.nil? %>
+            <%= sanitize @category.long_description.html_safe %>
+          <% else %>
+            <p>Page under construction, please check back soon or use the menu on the left to navigate the pages in this section.</p>
```

Comment:
We should localize this.

Reply:
It's pretty specific to categories, and temporary, are you sure you want this in config?